to.etc.domui/src/main/java/to/etc/domui/component/layout/TabPanelBase.java

```diff
@@ -428,15 +450,24 @@ public int getTabCount() {
 		return m_tablist.size();
 	}
 
-
-	public void setOnTabSelected(ITabSelected onTabSelected) {
+	public void setOnTabSelected(@Nullable ITabSelected onTabSelected) {
 		m_onTabSelected = onTabSelected;
 	}
 
+	@Nullable
 	public ITabSelected getOnTabSelected() {
 		return m_onTabSelected;
 	}
 
+	public void setOnTabSelectable(@Nullable ITabSelectable onTabSelectable) {
```

Comment:
Add a Javadoc here, and also tell there that the handler MUST give some kind of indication of WHY switching tabs is not possible, otherwise you will just have a thing that does not react when clicked which is a Bad Thing(tm).